crypto/whrlpool/wp_block.c

```diff
@@ -48,6 +48,12 @@ typedef unsigned long u64;
 typedef unsigned long long u64;
 #endif
 
+#ifdef __GNUC__
```

Comment:
This should be conditional on STRICT_ALIGNMENT. For same reason as in e,g. crypto/modes/cbc128.c cases. Because there is explicit check for pointer alignment.

Reply:
Ok, works, will place that in `#ifndef STRICT_ALIGMENT`